### PR TITLE
release: v1.0.0a1

### PR DESCRIPTION
**Pull request by omelkonian**
Signed-off-by: Orestis Melkonian <melkon.or@gmail.com>

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -24,6 +24,6 @@
 Changes
 =======
 
-Version 0.1.0 (released TBD)
+Version 1.0.0 (released December 15, 2016)
```
Comment (jirikuncar) [inline CHANGES.rst]: 1.0.0a1

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -19,12 +19,12 @@ What's new
 Installation
 ------------
 
-   $ pip install invenio-sequencegenerator==0.1.0
+   $ pip install invenio-sequencegenerator==1.0.0
```
Comment (jirikuncar) [inline RELEASE-NOTES.rst]: 1.0.0a1

File: invenio_sequencegenerator/version.py [hunk 3]
```diff
@@ -28,4 +28,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = "0.1.0.dev20160000"
+__version__ = "1.0.0.a1"
```
Comment (jirikuncar) [inline invenio_sequencegenerator/version.py]: 1.0.0a1

**Comment by egabancho** [comment]
LGTM

**Review by egabancho — APPROVED** [review]

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -19,12 +19,12 @@ What's new
...
 ------------
 
-   $ pip install invenio-sequencegenerator==0.1.0
+   $ pip install invenio-sequencegenerator==1.0.0a1
 
 Documentation
 -------------
 
-   http://invenio-sequencegenerator.readthedocs.io/en/v0.1.0
+   http://invenio-sequencegenerator.readthedocs.io/en/v1.0.0
```
Comment (jirikuncar) [inline RELEASE-NOTES.rst]: `1.0.0a1`

File: CHANGES.rst [hunk 5]
```diff
@@ -24,6 +24,6 @@
 Changes
 =======
 
-Version 0.1.0 (released TBD)
+Version 1.0.0a1 (released December 15, 2016)
```
Comment (jirikuncar) [inline CHANGES.rst]: needs an update

File: RELEASE-NOTES.rst [hunk 6]
```diff
@@ -19,12 +19,12 @@ What's new
 Installation
 ------------
 
-   $ pip install invenio-sequencegenerator==0.1.0
+   $ pip install invenio-sequencegenerator==1.0.0a1
```
Comment (jirikuncar) [inline RELEASE-NOTES.rst:22]: There are more versions and dates that should be updated (see the top of the file).

**Review by jirikuncar — CHANGES_REQUESTED** [review]

**Comment by omelkonian** [comment]
@jirikuncar Should I update the release date again?

**Review by jirikuncar — CHANGES_REQUESTED** [review]
Please add 2017 to copyright headers in touched files.

**Comment by omelkonian** [comment]
@jirikuncar done!

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**